tests/acceptance/stepDefinitions/accountContext.js

```diff
@@ -0,0 +1,25 @@
+const { client } = require('nightwatch-api')
+const { When, Then } = require('cucumber')
+const assert = require('assert')
+const _ = require('lodash')
+
+When('the user browses to the account page', function () {
+  return client
+    .page.accountPage()
+    .navigateAndWaitTillLoaded()
+})
+
+Then('the user should have following details displayed on the account information', async function (dataTable) {
+  const expectedAccInfo = dataTable.rowsHash()
+  const actualAccInfo = await client.page.accountPage().getAccountInformation()
+  const actualEntries = Object.entries(actualAccInfo)
+  for (const [key, value] of actualEntries) {
+    if (key === 'Groups membership') {
+      const differenceInGroups = _.difference(value.split(',').map(grp => grp.trim()),
+        expectedAccInfo[key].split(',').map(grp => grp.trim()))
```

Comment:
Please fix the formatting in a way it's more easy to follow.

Reply:
done